exporter_test.go

```diff
@@ -175,3 +175,55 @@ func TestParseUnixSocketAddress(t *testing.T) {
 		})
 	}
 }
+
+func TestParseConstLabels(t *testing.T) {
```

Comment:
I think if we you split the tests into TestParseConstLabels and TestParseConstLabelsFails, the body of test functions will be simplified - you will have either want, not both `want` and `wantErr` and only one condition in the loop

Reply:
I'm not a fan of this change. 

Within the `exporter_test.go` file the rest of the tests use the `wantErr` pattern. This would make this test inconsistent. 

I don't think it's complex to warrant making this test inconsistent with the rest. - What do you think?

Reply:
sounds good